src/operator/tensor/la_op.cc

```diff
@@ -227,10 +231,11 @@ NNVM_REGISTER_OP(_linalg_potri)
 .describe(R"code(Performs matrix inversion from a Cholesky factorization.
 Input is a tensor *A* of dimension *n >= 2*.
 
-If *n=2*, *A* is a lower triangular matrix (entries of upper triangle are all zero)
+If *n=2*, *A* is a triangular matrix (entries of upper or lower triangle are all zero)
 with positive diagonal. We compute:
 
-  *out* = *A*\ :sup:`-T` \* *A*\ :sup:`-1`
+  *out* = *A*\ :sup:`-T` \* *A*\ :sup:`-1` if *lower* = *true*
+  *out* = *A*\ :sup:`-1` \* *A*\ :sup:`-T` if *lower* = *false*
 
 In other words, if *A* is the Cholesky factor of a symmetric positive definite matrix
 *B* (obtained by *potrf*), then
```

Comment:
This is confusing, the example is *out* = *A*\ :sup:`-T` \* *A*\ :sup:`-1` so "out" is function of "A", and then here you refer to "B", whats "B"? Do you mean "out"? Can you use the same format as above:
*A* = *B* \* *B*\ :sup:`T`
 where A is the output and B is the input for consistency?

Reply:
The name of the input of the function is currently "A" in FListInputNames() so changing this would make the operator interface non-backward compatible. So we have to stick with it and then use also "A" accordingly in the documentation as the input to this function. Note that it is explained below that the output "out" is the inverse of a matrix "B" if the input "A" is the Cholesky-factor of "B". 
I tried a few rewordings, but none of them made this clearer.

Reply:
thanks for the explanation, makes sense.